app/models/rental.py

```diff
@@ -0,0 +1,13 @@
+from flask import current_app
+from app import db
+
+class Rental(db.Model):
+    rental_id = db.Column(db.Integer, primary_key=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('video.video_id'))
+    customer_id = db.Column(db.Integer, db.ForeignKey('customer.customer_id'))
+    due_date = db.Column(db.DateTime, nullable=True, default=None)
+    checked_out = db.Column(db.Boolean, default = True)
+
+    def check_in(self):
```

Comment:
Love to see all these little helper methods! 
One suggestion for improvement I think of is that this method could probably just go ahead and call `check_in` for Video and Customer, since that is part of the process of checking in every rental.